### DISCUSSION TITLE
Can I add my custom documents (like in gpt4all or private gpt)?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [oobabooga/text-generation-webui#3430] [General]
Can I add my custom documents (like in gpt4all or private gpt)?

**Comment by TomLucidor** [comment]
Maybe check PrivateGPT or PDFChat https://github.com/oobabooga/text-generation-webui/discussions/2057 https://github.com/oobabooga/text-generation-webui/discussions/1933
